src/PrestaShopBundle/Form/Admin/AdvancedParameters/AuthorizationServer/ApiAccessType.php

```diff
@@ -93,6 +95,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ]),
                 ],
             ])
+            ->add('lifetime', IntegerType::class, [
+                'label' => $this->trans('Lifetime', 'Admin.Global'),
+                'required' => false,
+                'empty_data' => '3600',
```

Comment:
For the default value this should be defined in the form data provider, but not blocking either